### PR TITLE
Create yant.csv

### PR DESCRIPTION
**Pull request by kintaro1981**
https://github.com/maxcleme/beadcolors/issues/48

### CODE REVIEW

File: raw/yant.csv [hunk 1]
```diff
@@ -0,0 +1,119 @@
+#02,white,255,255,255,KintaroOe
```
Comment (maxcleme) [inline raw/yant.csv:1]: Do you mind dropping that `#` or maybe using `Y` for "yant" maybe?

```suggestion
02,white,255,255,255,KintaroOe
```

```suggestion
Y02,white,255,255,255,KintaroOe
```

**Review by maxcleme — COMMENTED** [review]
Did you exclude P and L beads on purpose?

File: raw/yant.csv (hunk 1 above)
Comment (kintaro1981) [reply raw/yant.csv:1]: > Do you mind dropping that `#` or maybe using `Y` for "yant" maybe?

I simply used their numbering mode. I think there is no problem dropping the # or substitute it with the Y.

**Comment by kintaro1981** [comment]
> Did you exclude P and L beads on purpose?

yes, but I forgot to creat separated csv.
Let me know if it's preferable to include them in the "main" csv o create separated files

**Comment by maxcleme** [comment]
> > Did you exclude P and L beads on purpose?
> 
> yes, but I forgot to creat separated csv. Let me know if it's preferable to include them in the "main" csv o create separated files

I think we should include them in this CSV, yes

**Comment by kintaro1981** [comment]
- replaced # with Y
- added zeroes to have 3 digits reference codes

I double checked and I noticed that RGB values for P (Pearl) and L (glow in the dark) are not correct.
I think it's better to avoid them for now.

**Comment by maxcleme** [comment]
Dang, we forgot to update the README! 🙈 

Do you mind opening another PR when you'll have time? 🙇

**Pull request merged**